### DISCUSSION TITLE
Affliction nameidentifier does not work

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14745] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![602960_22](https://github.com/user-attachments/assets/fc08fdd4-9608-4ebf-bb99-2b5ece739489)

### Reproduction steps

1. change game language
2. giveaffliction slowparalysis 20

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming up in the next unstable build.
